### PR TITLE
[script.module.autocompletion@matrix] 2.1.0

### PR DESCRIPTION
**Pull request by finkleandeinhorn**
### Add-on details:

- **General**
  - Add-on name: AutoCompletion library
  - Add-on ID: script.module.autocompletion
  - Version number: 2.1.0
  - Kodi/repository version: matrix

- **Code location**
  - URL: https://github.com/finkleandeinhorn/script.module.autocompletion
  
Module providing some AutoCompletion functions

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Review by enen92 — COMMENTED** [review]
~Please check your pull requests, you submitted 2 PRs for the same version with different changesets.~
Nvm

File: script.module.autocompletion/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="script.module.autocompletion" name="AutoCompletion library" version="2.0.6" provider-name="Philipp Temminghoff (phil65), bbaronSVK, finkleandeinhorn, berkhornet">
+<addon id="script.module.autocompletion" name="AutoCompletion library" version="2.1.0" provider-name="Philipp Temminghoff (phil65), bbaronSVK, finkleandeinhorn, berkhornet">
```
Comment (enen92) [inline script.module.autocompletion/addon.xml:2]: Do you mind changing the Lang attributes to en_GB instead of en, while you're at it?

**Review by enen92 — APPROVED** [review]
Nvm my last comment, just realized this is the script module.

**Comment by finkleandeinhorn** [comment]
Don't merge this one quite yet, found a bug.

**Pull request merged**